src/camera-controls.js

```diff
@@ -542,8 +542,11 @@ export default class CameraControls {
 
 	setLookAt( position, target, enableTransition ) {
```

Comment:
how about just taking number literals?
```suggestion
	setLookAt( positionX, positionY, positionZ, targetX, targetY, targetZ, enableTransition ) {
```

Reply:
yeah first I thought about it, but I'd been afraid of argument count explosion.

Reply:
How about the following:

separate this into simpler methods. for the position, and for the target.
We could use the existing `moveTo` methods for the position and `lookat` to the target from the position.

thus, something like this
```
	setLookAt( targetX, targetY, targetZ, enableTransition ) {

		const _position = this.getPosition( _v3a );
		const _target = _v3b.set( targetX, targetY, targetZ );

		this._targetEnd.copy( _target );
		this._sphericalEnd.setFromVector3( _position.sub( _target ) );
		this._sanitizeSphericals();

		if ( ! enableTransition ) {

			this._target.copy( this._targetEnd );
			this._spherical.copy( this._sphericalEnd );

		}

		this._needsUpdate = true;

	}
```

then, users can do this
```
control.moveTo( 5, 6, 7, true );
control.setLookAt( 2, 2, 2, true );
```

Reply:
er, sorry forget about this, `moveTo` is not for the position...
but I prefer simpler functions like that